crates/oxc_linter/src/snapshots/no_access_key.snap

```diff
@@ -6,87 +6,87 @@ expression: no_access_key
   │ a11y complications.
    ╭─[no_access_key.tsx:1:1]
  1 │ <div accesskey="h" />
-   ·  ───
+   ·      ─────────────
    ╰────
   help: Remove the accessKey attribute.
 
   ⚠ eslint-plugin-jsx-a11y(no-access-key): No access key attribute allowed. Inconsistencies between keyboard shortcuts and keyboard commands used by screenreaders and keyboard-only users create
```

Comment:
Should the second sentence go into `help` instead?

Reply:
@Boshen 

That would be nice. so I fixed it!
Thank you for your review!